go/tasks/plugins/array/k8s/launcher.go

```diff
@@ -31,6 +21,7 @@ const (
 	ErrSubmitJob              errors2.ErrorCode = "SUBMIT_JOB_FAILED"
 	JobIndexVarName           string            = "BATCH_JOB_ARRAY_INDEX_VAR_NAME"
 	FlyteK8sArrayIndexVarName string            = "FLYTE_K8S_ARRAY_INDEX"
+	ResourcesPrimaryLabel     string            = "token"
```

Comment:
nit: should this be something like "current_cluster" or something?

Reply:
Actually this value isn't being used. The value specified as part of the config is being used. Removed.